driver/flambda_backend_args.ml

```diff
@@ -0,0 +1,757 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*              Damien Doligez, projet Para, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 1998 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+(* Keep in sync with tools/ocaml_compiler_internal_params. *)
```

Comment:
These don't appear to be in sync at the moment?
Also please document the exact rule about what needs to go in the file -- presumably it's only the ones that are parsed in `OCAMLPARAM`.  (Some are not e.g. `-drawfexpr`.)

Reply:
oh actually maybe they are in sync and I was misreading.  Can we generate the file somehow so we don't have this in-sync requirement?